CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to Rose
+
+## Prerequisites
+
+Make sure to have these tools installed:
+
+- [Git](https://git-scm.com/downloads)
+- [Rust](https://www.rust-lang.org/tools/install)
```

Comment:
Under list, could you also list that we need the `wasm32-unknown-unknown` Rust target, as well as specifically version 0.2.84 of the `wasm-bindgen-cli` crate installed globally?